lib/util/include/hse_util/rcu.h

```diff
@@ -1,36 +0,0 @@
-/* SPDX-License-Identifier: Apache-2.0 */
-/*
- * Copyright (C) 2015-2021 Micron Technology, Inc.  All rights reserved.
- */
-
-#ifndef HSE_PLATFORM_RCU_H
-#define HSE_PLATFORM_RCU_H
-
-/*
- * This snippet exists as a proxy to solve release buildtype issues due to
- * unread/unused variable warnings in urcu/wfcqueue.h.
- */
-
-#pragma GCC diagnostic push
-#pragma GCC diagnostic ignored "-Wunused-but-set-variable"
-#pragma GCC diagnostic ignored "-Wunused-variable"
-
-#include <urcu-bp.h>
-#include <urcu/rculist.h>
-
-#pragma GCC diagnostic pop
-
-#ifdef rcu_barrier
-#define HSE_HAVE_RCU_BARRIER 1
-#else
-#define rcu_barrier rcu_barrier_bp
-
-void
-rcu_barrier_bp(void);
-#endif
-
-#ifndef rcu_read_ongoing
```

Comment:
When is rcu_read_ongoing not defined?

Reply:
Old versions of liburcu for old distros that we no longer support.